crates/ocg_common/src/voxel/generator/mod.rs

```diff
@@ -312,23 +373,28 @@ impl StdGenerator {
             self.edges.push(rc);
         }
     }
-    
+
     /// returns: \[(delaunay edges, voronoi edges)\]
-    fn make_edges(voronoi: &Voronoi, points: &Vec<Point>) -> Vec<(PointEdge, PointEdge)> {
+    fn make_edges(voronoi: &Voronoi, points: &[Point]) -> Vec<(PointEdge, PointEdge)> {
         let mut list_of_delaunay_edges: Vec<PointEdge> = vec![];
-    
+
         let triangles = &voronoi.triangulation().triangles;
-        let triangles: Vec<[&Point; 3]> = (0..triangles.len() / 3).map(|t| [
-            &points[triangles[3 * t + 0]],
-            &points[triangles[3 * t + 1]],
-            &points[triangles[3 * t + 2]]]
-        ).collect();
+        let triangles: Vec<[&Point; 3]> = (0..triangles.len() / 3)
+            .map(|t| {
+                [
+                    &points[triangles[3 * t]],
```

Comment:
I thought the `t + 0` looked cleaner.

Reply:
It does look cleaner, but then I either have to add a linter suppression rule for this line, and I'd rather not disable warning about useless arithmetic operations globally as that catches some bugs.

Reply:
👍